exporter/coralogixexporter/client.go

```diff
@@ -70,12 +65,20 @@ func (c *coralogixClient) newPost(ctx context.Context, td pdata.Traces) error {
 	if err != nil {
 		return fmt.Errorf("can't translate to jaeger proto: %w", err)
 	}
+
 	ctx = metadata.NewOutgoingContext(ctx, metadata.New(c.cfg.GRPCClientSettings.Headers))
 	for _, batch := range batches {
-
-		_, err := c.client.PostSpans(ctx, &cxpb.PostSpansRequest{Batch: *batch, Metadata: &c.metadata}, grpc.WaitForReady(false))
-		if err != nil {
-			return fmt.Errorf("failed to push trace data via Coralogix exporter %w", err)
+		if batch.GetProcess().GetServiceName() != "" {
+			_, err := c.client.PostSpans(ctx, &cxpb.PostSpansRequest{
+				Batch:    *batch,
+				Metadata: &cxpb.Metadata{ApplicationName: c.cfg.AppName, SubsystemName: batch.GetProcess().GetServiceName()},
+			}, grpc.WaitForReady(c.cfg.WaitForReady))
+			if err != nil {
+				return fmt.Errorf("Failed to push trace data via Coralogix exporter %w", err)
```

Comment:
Please, follow the Go conventions by not uppercasing the first letter from the error message

Reply:
I changed the uppercase to lowercase according to the conventions.